extensions/spark/kyuubi-spark-authz/src/main/resources/table_command_spec.json

```diff
@@ -2056,6 +2056,30 @@
     "fieldExtractor" : "LogicalPlanQueryExtractor"
   } ],
   "uriDescs" : [ ]
+}, {
+  "classname" : "org.apache.spark.sql.delta.commands.OptimizeTableCommand",
```

Comment:
can you provide us the link to this class？

Reply:
https://github.com/delta-io/delta/blob/master/spark/src/main/scala/org/apache/spark/sql/delta/commands/OptimizeTableCommand.scala